pkg/scheduler/plugins/overcommit/overcommit.go

```diff
@@ -144,3 +218,42 @@ func (op *overcommitPlugin) OnSessionClose(ssn *framework.Session) {
 	op.idleResource = nil
 	op.inqueueResource = nil
 }
+
+// parseFactor is used to parse the value corresponding to the given key and store the result
+// in the float64 variable pointed to by target.
+// If the specified key exists in pluginArguments, the corresponding value is stored in target;
+// if the specified key does not exist but the default overCommitFactor does,
+// its corresponding value is stored in target.
+func (op *overcommitPlugin) parseFactor(key string, target *float64) {
+	if op.pluginArguments.KeyExists(key) {
```

Comment:
Why need check exist? Get method has handled it。

Reply:
My idea at the time was that we need to know clearly whether this key is used in the configuration, so that we can judge the subsequent logic.
If we only use the original method, I don't know whether this value is the default value or the configuration file setting

Reply:
The priority of these fields will be from low to high:

defaultOverCommitFactor -> overcommit-factor -> cpu-overcommit-factor, men-overcommit-factor, other overcommit-factor

Reply:
fixed